pykafka/simpleconsumer.py

```diff
@@ -408,37 +410,50 @@ def _handle_success(parts):
             to_retry.extend(parts_by_error.get(NotCoordinatorForConsumer.ERROR_CODE, []))
             reqs = [p.build_offset_fetch_request() for p, _ in to_retry]
 
-    def _reset_offsets(self, partitions=None):
-        """Reset offsets after an error
+    def reset_offsets(self, partitions=None):
+        """Reset offsets for the specified partitions
 
         Issue an OffsetRequest for each partition and set the appropriate
-        returned offset in the OwnedPartition per self._auto_offset_reset
+        returned offset in the OwnedPartition
 
-        :param partitions: the partitions for which to reset offsets
+        :param partitions: (`partition`, `offset`) pairs to reset
+            where `partition` is the partition for which to reset the offset
+            and `offset` is the new offset the partition should have
         :type partitions: Iterable of
-            :class:`pykafka.simpleconsumer.OwnedPartition`
+            (:class:`pykafka.simpleconsumer.OwnedPartition`, int)
```

Comment:
I see what you mean that that feels inconsistent, but on the other hand as a lazy user of the interface I'd say that the nice thing about ids is that you can save them to disk, and so when you get them from disk, you could pass them straight in here.

Maybe I'm missing a trick though, and there's a shortcut somehow. Am I right in thinking you'd have to get the ids, then map them to `partition.Partition`s (which is what you get from the `BalancedConsumer.partitions` property) and then again map to `simpleconsumer.OwnedPartition`s to call `reset_offsets`?